temba/channels/handlers.py

```diff
@@ -207,6 +207,142 @@ def post(self, request, *args, **kwargs):
         return HttpResponse("Not Handled, unknown action", status=400)
 
 
+class TwimlAPIHandler(View):
```

Comment:
This looks like a giant copy/paste, surely it would be possible to abstract this out with the other TwilioHandler so that we aren't repeating so much code.